admin-console/src/main/webapp/src/components/EventTypes/CreateEditModal.tsx

```diff
@@ -16,12 +19,19 @@ interface CreateEditModalProps {
 export const CreateEditModal: React.FunctionComponent<CreateEditModalProps> = (props) => {
 
     const [ eventType, setEventType ] = React.useState<Partial<EventType>>(props.initialEventType ?? {});
+    const [ systemBehaviorGroup, setSystemBehaviorGroup ] = React.useState<Partial<BehaviorGroup>>();
+    const getBehaviorGroups = useSystemBehaviorGroups();
 
     const handleChange = (value: string, event: React.FormEvent<HTMLInputElement> | React.FormEvent<HTMLTextAreaElement>) => {
         const target = event.target as HTMLInputElement;
         setEventType(prev => ({ ...prev, [target.name]: target.value }));
     };
 
+    const handleSelect = (value: string, event: React.FormEvent<HTMLSelectElement>) => {
```

Comment:
would this be where I need to utilize the linking BG api?

Reply:
No, I think you would do that on the `onSave`.